lib/clearance/authorization.rb

```diff
@@ -3,15 +3,21 @@ module Authorization
     extend ActiveSupport::Concern
 
     included do
-      hide_action :authorize, :deny_access
+      hide_action :authorize, :deny_access, :require_login
     end
 
-    def authorize
+    def require_login
       unless signed_in?
         deny_access
       end
     end
 
+    def authorize
+      warn "[DEPRECATION] Clearance's `authorize` before_filter is deprecated."\
```

Comment:
Updated the deprecation to use concatenation and be more explicit about the potential gotcha with `skip_before_filter`